README.md

```diff
@@ -29,13 +30,12 @@ To get a local copy up and running follow these simple example steps.
 ## Author
 
 GitHub:
-[@sntgopl](https://github.com/sntgopl)
 [@Mov305](https://github.com/Mov305)
-Linkedin: Santiago Pons Lezica & Abdelrhman samy
+Linkedin: Abdelrhman samy
 
```

Comment:
- [ ] Please add your social media handles in proper and clickable links as shown in the [sample](https://github.com/HAMMAS-SALEEM/ToDoList/blob/main/README.md#authors). It makes your accounts easily accessible to recruiters and viewers of your projects :thinking: